### PR TITLE
Updates font sizes.

### PR DESCRIPTION
**Pull request by jeffikus**
#### Changes proposed in this Pull Request:

Updates the font sizes to match the Design file changes from @beafialho to Quadrat design.

#### Related issue(s):

#4683 

**Test Plan:**

**Test that the font sizes on Desktop are:**
h1: 80px
h2: 65px
h3: 38px
h4: 22px
h5: 20px
h6: 18px
P Tiny: 16px
P Small: 18px
P Normal: 20px
P Large: 22px
P Extra Large: 38px
P Huge: 48px
Site Title: 20px
Site Tag: 16px

**Test that the font sizes on Mobile are:**
h1: 48px
h2: 36px
h3: 28px
h4: 22px
h5: 20px
h6: 18px
P Tiny: 16px
P Small: 18px
P Normal: 20px
P Large: 22px
P Extra Large: 28px
P Huge: 36px

Test that Quote Citation font size is: 16px
Test that Query Pagination font size is: 20px

### CODE REVIEW

**Review by danieldudzic — COMMENTED** [review]
LGTM, but since it's late - I think a second pair of eyes would be great :)

**Review by scruffian — DISMISSED** [review]

File: geologist/child-theme.json [hunk 1]
```diff
@@ -244,14 +244,14 @@
...
-					"fontSize": "20px",
+					"fontSize": "var(--wp--preset--font-size--normal)",
 					"fontWeight": "400",
 					"lineHeight": "1.6"
 				}
 			},
 			"core/navigation": {
 				"typography": {
-					"fontSize": "18px"
+					"fontSize": "var(--wp--preset--font-size--normal)"
```
Comment (scruffian) [inline geologist/child-theme.json:254]: Should this one be small?

File: geologist/child-theme.json [hunk 2]
```diff
@@ -317,14 +317,14 @@
...
-							"fontSize": "min(max(28px, 5vw), 38px)"
+							"fontSize": "var(--wp--preset--font-size--extra-large)"
 						}
 					}
 				}
 			},
 			"core/query-pagination": {
 				"typography": {
-					"fontSize": "18px",
+					"fontSize": "var(--wp--preset--font-size--normal)",
```
Comment (scruffian) [inline geologist/child-theme.json:327]: Should this one be small?

File: geologist/child-theme.json [hunk 3]
```diff
@@ -333,7 +333,7 @@
 					"link": "var(--wp--custom--color--primary)"
 				},
 				"typography": {
-					"fontSize": "18px",
+					"fontSize": "var(--wp--preset--font-size--normal)",
```
Comment (scruffian) [inline geologist/child-theme.json:336]: Should this one be small?

File: geologist/child-theme.json (hunk 1 above)
Comment (scruffian) [inline geologist/child-theme.json:254]: Should this one be small?

File: geologist/child-theme.json [hunk 4]
```diff
@@ -357,28 +357,28 @@
...
-					"fontSize": "min(max(28px, 5vw), 38px)",
+					"fontSize": "var(--wp--preset--font-size--extra-large)",
 					"lineHeight": 1.2
 				}
 			},
 			"h4": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "20px",
+					"fontSize": "var(--wp--preset--font-size--large)",
```
Comment (scruffian) [inline geologist/child-theme.json:367]: Should this one be normal?

File: geologist/child-theme.json [hunk 5]
```diff
@@ -357,28 +357,28 @@
...
 					"lineHeight": 1.4
 				}
 			},
 			"h5": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "18px",
+					"fontSize": "var(--wp--preset--font-size--normal)",
```
Comment (scruffian) [inline geologist/child-theme.json:374]: Should this one be small?

File: geologist/child-theme.json [hunk 6]
```diff
@@ -357,28 +357,28 @@
...
 					"lineHeight": 1.4
 				}
 			},
 			"h6": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "16px",
+					"fontSize": "var(--wp--preset--font-size--small)",
```
Comment (scruffian) [inline geologist/child-theme.json:381]: Should this one be tiny?

File: geologist/child-theme.json (hunk 2 above)
Comment (jeffikus) [reply geologist/child-theme.json:327]: No, in Quadrat I first updated these to 20px then to the variable. Here I scaled it up correctly to the variable. See https://github.com/Automattic/themes/pull/4683#discussion_r714278294

File: geologist/child-theme.json (hunk 3 above)
Comment (jeffikus) [reply geologist/child-theme.json:336]: No, same as the above comment, for Site Title - same as Quadrat https://github.com/Automattic/themes/pull/4683#discussion_r714278696

File: geologist/child-theme.json (hunk 1 above)
Comment (jeffikus) [reply geologist/child-theme.json:254]: Quadrat is also normal. Same as the above comments.

File: geologist/child-theme.json (hunk 4 above)
Comment (jeffikus) [reply geologist/child-theme.json:367]: Same as the above. Quadrat - https://github.com/Automattic/themes/pull/4683/files#diff-a1fd00442cdce71e5586a4cc5607a542027fe9cbc72ab9aa6f182d5b8f524142R392

File: geologist/child-theme.json (hunk 5 above)
Comment (jeffikus) [reply geologist/child-theme.json:374]: Same as the above. Quadrat - https://github.com/Automattic/themes/pull/4683/files#diff-a1fd00442cdce71e5586a4cc5607a542027fe9cbc72ab9aa6f182d5b8f524142R399

File: geologist/child-theme.json (hunk 6 above)
Comment (jeffikus) [reply geologist/child-theme.json:381]: Same as the above. Quadrat - https://github.com/Automattic/themes/pull/4683/files#diff-a1fd00442cdce71e5586a4cc5607a542027fe9cbc72ab9aa6f182d5b8f524142R406

**Review by scruffian — APPROVED** [review]

**Comment by scruffian** [comment]
Deployed in D67367-code

**Pull request merged**